src/bootstrap/flags.rs

```diff
@@ -63,6 +74,8 @@ impl Flags {
     pub fn parse(args: &[String]) -> Flags {
         let mut opts = Options::new();
         opts.optflag("v", "verbose", "use verbose output");
+        opts.optflag("", "very-verbose", "use very verbose output");
```

Comment:
Normally this is interpreted as `-vv`, but I'm not sure if getopts has the ability to tell us about that?